docs/plugin_development.rst

```diff
@@ -701,237 +796,263 @@ Finally, we add the following method to our plugin:
-
        ID = 'pomodoro_timer_toolbar'
 
-As you can see, this statement is very simple. It only needs to declare an ``ID``, that serves to identify our toolabr among the rest.
+The only customization we need is to declare an ``ID`` to identify our particular toolbar.
 
-It is possible to include other Qt widgets in our toolbar, but in this case it's better to use the appropriate Spyder methods for that in order to maintain their relationship with the rest of the application. In other words, as long as the widget you need exists in ``spyder.api.widgets``, use it!
-
-
-Next, we need to declare a boolean variable in our status widget to indicate if the countdown is paused or not. For that, let's add the following inside the ``__init__`` method of ``PomodoroTimerStatus``:
+Next, we need to declare a boolean variable in our status widget to indicate if the countdown is paused or not.
+Let's add the following code inside the ``__init__`` method of the ``PomodoroTimerStatus`` class definition:
 
 .. code-block:: python
 
    self.pause = True
 
-And inside the ``update_timer`` method, substitute
+And inside the ``update_timer`` method of the same class, replace
 
 .. code-block:: python
 
            if self.countdown > 0:
-               ...
 
-by
+with
 
 .. code-block:: python
 
            if self.countdown > 0 and not self.pause:
-               ...
 
+Here's a full listing of the `TIMER -> TOOLBAR widgets.py diff <https://github.com/spyder-ide/spyder-pomodoro-timer/commit/48a946fdbb934b9b85facd3c1b77fc8999e049a9>`__.
 
-Create the Pomodoro Toolbar
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Now we are going to create a new section in our toolbar and associate some functionality to it by means of actions. This particular information is recommended to be included in the ``api.py`` file because this way we can offer endpoints to the rest of Spyder and new plugins for tweaking the behavior of our plugin.
+.. _tutorial-toolbar-actions:
 
-`TIMER -> ACTIONS api.py diff`_
+Declaring the high-level toolbar structure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. _TIMER -> ACTIONS api.py diff: https://github.com/map0logo/spyder-pomodoro-timer/commit/cf540f972f37aaf5d6ccc8524cbcc7aeae9c483b
+Now we'll declare the high-level structure of our new toolbar, including its actions and sections, as well as that of a menu for our plugin.
+This goes in the ``api.py`` file, since that allows Spyder and other plugins to modify and add to our toolbar if needed.
 
-Let's add the following to the end of ``api.py``:
+.. note:: Actions represent the control buttons that have associated functions to be called when interacted with, while toolbar sections contain groups of actions.
+
+Let's add the following to ``api.py``:
 
 .. code-block:: python
 
    class PomodoroToolbarActions:
-       Start = 'start_timer'
-       Pause = 'pause_timer'
-       Stop = 'stop_timer'
-
+       Start = "start_timer"
+       Pause = "pause_timer"
+       Stop = "stop_timer'
```

Comment:
Mismatched closing quote here

Reply:
Thanks for noticing it @merwok! Could you submit a pull request to fix that?

Reply:
I don’t think I’ll have the time